src/features/CourtCaseList/NotePreviewButton.tsx

```diff
@@ -0,0 +1,79 @@
+import ConditionalRender from "../../components/ConditionalRender"
+import { Dispatch, SetStateAction } from "react"
+import { createUseStyles } from "react-jss"
+
+const useStyles = createUseStyles({
+  buttonContainer: {
+    color: "#1d70b8",
+    width: "97px"
+  },
+  notePreviewContainer: {
+    backgroundColor: "#F3F2F1",
```

Comment:
`#F3F2F1` is used a few times in the codebase. We could save it in src/utils/colours.ts as a variable & reuse.

Reply:
Good idea- thanks.